src/components/HTMLEngineProvider/types.ts

```diff
@@ -1,13 +1,7 @@
-import {ReactNode} from 'react';
-import {TNode} from 'react-native-render-html';
+import type {TNode} from 'react-native-render-html';
+import type ChildrenProps from '@src/types/utils/ChildrenProps';
 
-type HTMLEngineProviderProps = {
-    /** Children to wrap in HTMLEngineProvider. */
-    children?: ReactNode;
-
-    /** Optional debug flag. Prints the TRT in the console when true. */
-    debug?: boolean;
-};
+type HTMLEngineProviderProps = ChildrenProps;
```

Comment:
We don't need to define a type that is just another type. We can just use `ChildrenProps` and remove `HTMLEngineProviderProps`